packages/core/src/components/TextField/TextField.scss

```diff
@@ -6,11 +6,15 @@ Text field
 
 @uswds https://standards.usa.gov/components/form-controls/#text-input
 
-Text fields allow users to enter any combination of letters, numbers, or symbols of their choosing (unless otherwise restricted). Text fields can span single or multiple lines.
+Text fields allow users to enter any combination of letters, numbers, or symbols of their choosing (unless otherwise restricted). Text fields can span single or multiple lines. Use the `ds-c-field--tiny` or `ds-c-field--medium` classes to constrain the width of the input.
```

Comment:
Should we include this in the description, or are the examples and code snippets enough for this?